example/example-graph/src/main/resources/example/gettingstarted/9/walkthrough.md

```diff
@@ -0,0 +1,54 @@
+${HEADER}
+
+${CODE_LINK}
+
+This example demonstrates how we can use a HyperLogLogPlus object to estimate the cardinality of vertices in a graph.
```

Comment:
Can we be more precise here? It should say "This example demonstrates how storing a HyperLogLogPlus object on each vertex in a graph allows us to quickly estimate its degree (i.e. the number of edges it is involved in). The estimate can be obtained without scanning through all the edges involving a node and so is very quick, even if the degree is very large."